src/components/button/Button.tsx

```diff
@@ -66,6 +68,12 @@ const IconOnly = styled.span`
   font-size: ${styleVariables.fontSizeM};
 `;
 
+const IconButon = styled.span`
```

Comment:
Het IconButon styledcomponent is juist hetzelfde als IconOnly, beter deze hergebruiken.